packit_service/worker/handlers/abstract.py

```diff
@@ -112,6 +112,14 @@ def _clean_workplace(self):
             else:
                 shutil.rmtree(item)
 
+    def pre_check(self) -> bool:
+        """
+        Validate the input.
+
+        :return: False if we can skip the run.
```

Comment:
pre_cache -> False mean that run() will not be executed?

Reply:
Currently only for the configured jobs (in the `process_jobs` method), not the hardcoded ones in the `process_message`. Shall I use that also there?

Reply:
I'd love to see this text in the docstring - the present docstring is really sparse.

Reply:
I've updated the doc string in #451

Reply:
I forgot to push that commit, will be in the next one.